website/content/docs/auth/saml.mdx

```diff
@@ -0,0 +1,158 @@
+---
+layout: docs
+page_title: SAML - Auth Methods
+description: |-
+  The "saml" auth method allows users to authenticate with Vault using their
+  identity in a SAML Identity Provider.
+---
+
+# SAML Auth Method
+
+<EnterpriseAlert />
+
+The `saml` auth method allows users to authentication with Vault using their identity
+within a [SAML V2.0](https://saml.xml.org/saml-specifications) Identity Provider.
+Authentication is suited for human users by requiring interaction with a web browser.
+
+## Authentication
```

Comment:
I chose these three second-level headings (Authentication, Configuration, API) because they're most consistent with the rest of our auth method documentation. My understanding is that purpose of the "Authentication" section is to provide details on how to authenticate using the auth method. This is targeted at a different persona than the "Configuration" section.

Would it be okay to keep this consistent with the majority of the Vault auth method docs?